app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ManageProspectCareTeam.php

```diff
@@ -94,21 +103,27 @@ public function table(Table $table): Table
                     ->modalSubmitActionLabel('Add')
                     ->attachAnother(false)
                     ->color('primary')
-                    ->recordSelect(
-                        fn (Select $select) => $select->placeholder('Select Users'),
-                    )
-                    ->multiple()
-                    ->recordSelectOptionsQuery(
-                        fn (Builder $query) => $query->tap(new HasLicense(Prospect::getLicenseType())),
-                    )
+                    ->mountUsing(fn (ComponentContainer $form) => $form->fill([
+                        'care_team_role_id' => CareTeamRoleType::prospectDefault()?->id,
```

Comment:
We like to use `getKey()` where ever we can due to some of our Models not using `id` as the primary key.

```suggestion
                        'care_team_role_id' => CareTeamRoleType::prospectDefault()?->getKey(),
```